document/core/appendix/custom.rst

```diff
@@ -68,7 +68,7 @@ Name Maps
 .........
 
 A *name map* assigns :ref:`names <syntax-name>` to :ref:`indices <syntax-index>` in a given :ref:`index space <syntax-index>`.
-It consists of a :ref:`vector <binary-vec>` of index/name pairs in arbitrary order.
+It consists of a :ref:`vector <binary-vec>` of index/name pairs in order of increasing index value.
```

Comment:
See Wikipedia, too. But I'm not sure how that is even relevant to the wording here -- "in order of increasing value" is common and unambiguous usage, while nobody says "in order of monotonically increasing value".